### DISCUSSION TITLE
Why chartjs-plugin-zoom does not work for Treemap library?

### DISCUSSION THREAD

**Discussion by mirzaeianfar** [kurkle/chartjs-chart-treemap#71] [Q&A] [answered]
When i add chartjs-plugin-zoom library for treemap ,it does not work for zooming.
How can i zoom in treemap??

**Comment by kurkle** [comment]
It is because treemap is not using any scales, but draws directly to to chartArea.

**Reply by mirzaeianfar** [reply]
Can you please tell me how I can zoom in on the treemap.
I desperately need to be able to zoom in on the treemap.

**Reply by kurkle** [reply]
Its not possible with the current implementation

**Reply by kurkle** [reply]
I have already created an issue to track this: #75

**Reply by mirzaeianfar** [reply]
I'm in a bit of a hurry, it 's not possible to guide me to read this myself on the current tool?
Or if it takes less than ten days, wait?

**Reply by stockiNail** [reply]
See PR https://github.com/kurkle/chartjs-chart-treemap/pull/141

**Comment by stockiNail** [comment, marked as answer]
Feature has been released in version [2.1.0](https://github.com/kurkle/chartjs-chart-treemap/releases/tag/v2.1.0)
